narration/generator.py

```
@@ -0,0 +1,456 @@
+
+            # DEBUG: Log which model and settings are being used
+            logger.debug(
+                f"🎭 NARRATION MODEL DEBUG: Using model '{self.model}' with max_tokens={self.max_tokens}"
+            )
+            logger.debug(
+                f"🎭 EXACT API MODEL NAME: '{self.model}' (checking for full GPT-5 vs nano)"
+            )
+
+            # Call LLM with appropriate API
+            if self.model.startswith("gpt-5"):
+                logger.debug(f"🚀 Using GPT-5 Responses API with text verbosity=medium")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)